sdk/core/azure-core-serializer-json-jackson/src/main/java/com/azure/core/serializer/json/jackson/JacksonJsonSerializer.java

```diff
@@ -93,16 +95,28 @@ public String convertMemberName(Member member) {
 
         if (member instanceof Method) {
             Method m = (Method) member;
+            String methodNameWithoutJavaBeans = removePrefix(m.getName());
             if (m.isAnnotationPresent(JsonIgnore.class)) {
                 return null;
             }
             if (m.isAnnotationPresent(JsonProperty.class)) {
                 String propertyName = m.getDeclaredAnnotation(JsonProperty.class).value();
-                return CoreUtils.isNullOrEmpty(propertyName) ? m.getName() : propertyName;
+                return CoreUtils.isNullOrEmpty(propertyName) ? methodNameWithoutJavaBeans : propertyName;
             }
-            return member.getName();
+            return methodNameWithoutJavaBeans;
         }
 
         return null;
     }
+
+    private String removePrefix(String methodName) {
+        if (methodName == null || !(methodName.contains("get") || methodName.contains("set"))) {
+            return methodName;
+        }
+        String noPrefix = methodName.replaceFirst("^get|set", "");
```

Comment:
Regex should be `"^get|^set"`. Current regex will replace `isPreset` to ``isPre`.

```java
jshell> System.out.println("isPreset".replaceFirst("^get|set", ""))
isPre

```

Reply:
Use BeanUtil in jackson lib to do the check